utils/automation/FoScanner/Scanners.py

```diff
@@ -157,6 +157,61 @@ def get_copyright_list(self, all_results: bool = False) \
       return copyright_list
     return False
 
+  def get_copyright_whole(self, all_results: bool = False) \
+      -> Union[List[ScanResult], bool]:
+    """
+    Get the formatted results from copyright scanner
+
+    :param all_results: Get all results even excluded files?
+    :type all_results: bool
+    :return: list of findings
+    :rtype: list[ScanResult]
+    """
+    copyright_results = self.__get_copyright_results()
+    copyright_list = list()
+    for result in copyright_results:
+      path = self.__normalize_path(result['file'])
+      if self.cli_options.repo is True and all_results is False and \
+          self.is_excluded_path(path) is True:
+        continue
+      if result['results'] is not None and result['results'] != "Unable to " \
+                                                                "read file":
+        contents = list()
+
+        for finding in result['results']:
+          if finding is not None and finding['type'] == "statement":
+            contents.append(finding)
+        if len(contents) > 0:
+          copyright_list.append(ScanResult(path, result['file'], contents))
```

Comment:
`contents` here is breaking the datatype hint of 3rd parameter of `ScanResult`. It expects `set[str]` but getting `list[dict[str,str]]`

Reply:
Hints are put in place to suggest what kind of data I can expect here.

Reply:
I'm on it. I'll correct it.